src/de/read.rs

```diff
@@ -101,7 +105,7 @@ impl<'storage> SliceReader<'storage> {
     }
 }
 
-impl<'storage> BincodeRead<'storage> for SliceReader<'storage> {
+impl<'a, 'storage> BincodeRead<'storage> for &'a mut SliceReader<'storage> {
```

Comment:
Why are these changes necessary?

Reply:
I guess now that we use `pub(crate)` throughout the coebase I can use the other option I outlined in my PR. Guess that happens when you don't update a PR for two years lol